sklearn/svm/classes.py

```diff
@@ -442,6 +442,11 @@ class frequencies.
     max_iter : int, optional (default=-1)
         Hard limit on iterations within solver, or -1 for no limit.
 
+    compact_decision_function : bool, default=False
+        Whether to return a decision function of shape (n_samples, n_classes)
+        as all other classifiers, or the original one-vs-one decision function
+        of libsvm which has shape (n_samples, n_classes * (n_classes - 1) / 2).
+
```

Comment:
The default is not actually `False`, it's `None`. That's going to be confusing for new users; please document the entire tribool.